app/lambda/list_sessions/lambda_function.py

```diff
@@ -16,9 +37,9 @@ def getItems(self, PatientId, HealthCareProfessionalId, SessionId):
 
     def handle(self, event, context):
         print("event: {}".format(event))
-        PatientId = event[DATASTORE_COLUMN_PATIENT_ID] if DATASTORE_COLUMN_PATIENT_ID in event else None
-        HealthCareProfessionalId = event[DATASTORE_COLUMN_HEALTH_CARE_PROFESSSIONAL_ID] if DATASTORE_COLUMN_HEALTH_CARE_PROFESSSIONAL_ID in event else None
-        SessionId = event[DATASTORE_COLUMN_SESSION_ID] if DATASTORE_COLUMN_SESSION_ID in event else None
+        PatientId = event["queryStringParameters"][DATASTORE_COLUMN_PATIENT_ID] if DATASTORE_COLUMN_PATIENT_ID in event["queryStringParameters"] else None
```

Comment:
Do we need this if else case?
Is there any scenario where we wont have patient/hp in list sessions?

Reply:
Yes. We do need the if else case, because there would be scenarios where patient/hp is not provided(search by patient id and session id or search by hp id and session id).

Reply:
Thanks!